combineFields.js

```diff
@@ -0,0 +1,57 @@
+const fs = require('fs');
+const path = require('path');
+const yaml = require('yaml');
+
+function combineFields(filePath) {
```

Comment:
Is this file for the automated changes? If so there is no need to include in the PR.

Reply:
This is because this would be a one-time data change

Reply:
Hi Sir,

Thank you for your feedback! I understand your point regarding the combineFields.js script being unnecessary for a one-time data change. I’ll try to implement the logic directly within gatsby-node.js itself to handle the field transformation during the build process.

I’ll update the PR soon and let you know once the changes are made.